docs/features.rst

```diff
@@ -117,20 +117,40 @@ The profile ``utils_cell_annotate`` should be added when generating the main con
     params {
         sc {
             cell_annotate {
-                iff = '10x_cellranger_mex'
                 off = 'h5ad'
+                method = ''
                 cellMetaDataFilePath = ''
+                sampleSuffixWithExtension = ''
                 indexColumnName = ''
                 sampleColumnName = ''
                 annotationColumnNames = ['']
             }
         }
     }
 
-Then, the following parameters should be updated to use the module feature:
+Two methods (``params.sc.cell_annotate.method``) are available:
+
+- ``aio``
+- ``obo``
+
+If you have a single file containing the metadata information of all your samples, use ``aio`` method otherwise use ``obo``.
+
+For both methods, here are the mandatory params to be set:
 
+- ``off`` should be set to ``h5ad``
+- ``method`` choose either ``obo`` or ``aio``
 - ``cellMetaDataFilePath`` is a TSV file (with header) with at least 2 columns: a column containing all the cell IDs and an annotation column.
-- ``indexColumnName`` is the column name from ``cellMetaDataFilePath`` containing the cell IDs information.
+
+If ``obo`` is used, the following params are required:
+
+- ``sampleSuffixWithExtension`` is the suffix used to extract the sample ID from the file name of ``cellMetaDataFilePath`` i.e.: it is the suffix after the sample name in the file path.
+- ``indexColumnName`` is the column name from ``cellMetaDataFilePath`` containing the cell IDs information. This column **should** have unique values. 
```

Comment:
Is this should have unique, or must have unique?

Reply:
`must` is a better word :+1: